scanpointgenerator/core/generator.py

```diff
@@ -22,14 +22,6 @@ class Generator(object):
     _generator_lookup = {}
     axes = []
 
-    def iterator(self):
```

Comment:
CompoundGenerator wouldn't implement it.
Perhaps we should more clearly split regular generators from CompoundGenerator - it behaves really quite differently now.